test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -154,21 +156,70 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-servic
+            bootstrap-path (format "%s,%s" bootstrap-one bootstrap-two)
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              (format "Loading bootstrap configs:\n%s\n%s"
+                      (fs/absolute bootstrap-one)
+                      (fs/absolute bootstrap-two))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing multiple .cfg files can be specified on the command line"
+    (with-test-logging
+      (let [bootstrap-path "./dev-resources/bootstrapping/cli/split_bootstraps/both/"
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              ; We can't know what order it will find the files on disk, so just
+              ; look for a partial match with the path we gave TK.
+              (re-pattern (format "Loading bootstrap configs:\n%s"
+                                  (fs/absolute bootstrap-path)))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing both a file and a folder can be specified on the command line"
+    (with-test-logging
+      (let [bootstrap-one-dir "./dev-resources/bootstrapping/cli/split_bootstraps/one/"
+            bootstrap-one "./dev-resources/bootstrapping/cli/split_bootstraps/one/bootstrap_one.cfg"
+            bootstrap-two "./dev-resources/bootstrapping/cli/split_bootstraps/two/bootstrap_two.cfg"
+            bootstrap-path (format "%s,%s" bootstrap-one-dir bootstrap-two)
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              (format "Loading bootstrap configs:\n%s\n%s"
+                      (fs/absolute bootstrap-one)
+                      (fs/absolute bootstrap-two))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world"))))))
+
 (deftest bootstrap-path-with-spaces
   (testing "Ensure that a bootstrap config can be loaded with a path that contains spaces"
     (with-test-logging
-      (let [app             (bootstrap-with-empty-config
-                               ["--bootstrap-config" "./dev-resources/bootstrapping/cli/path with spaces/bootstrap.cfg"])
-            test-svc        (get-service app :TestService)
+      (let [bootstrap-path "./dev-resources/bootstrapping/cli/path with spaces/bootstrap.cfg"
```

Comment:
It might also be good to have a variation of this test that has two bootstrap files where each file has spaces in the name - just to confirm that that split we do on file names doesn't adversely affect the ability to use any individual files that had spaces in the name.